src/components/Dashboard/ProprietorPartnersListSm.vue

```diff
@@ -0,0 +1,190 @@
+<template>
```

Comment:
This component is very similar to DirectorListSm except:
- its id is different
- it displays email address
- the list it displays is proprietor or partners